webui/react/src/pages/ProjectDetails.settings.ts

```diff
@@ -64,6 +65,12 @@ export interface ProjectDetailsSettings extends InteractiveTableSettings {
 
 const config: SettingsConfig = {
   settings: [
+    {
+      defaultValue: [],
+      key: 'pinned',
+      storageKey: 'pinned',
+      type: { baseType: BaseType.Integer, isArray: true },
+    },
```

Comment:
nit: I think this should probably have `skipUrlEncoding: true,`

Reply:
can i ask you a reason why?

Reply:
It makes the URL pretty long, but more importantly it would be very weird if someone sent you a link to a project that ended up overwriting your pinned experiments.

Reply:
Ah I thought its for avoiding encoding the data. Its for avoiding preserving setting in the URL query param.
That makes sense.
Thanks